packages/serverpod/lib/src/database/expressions.dart

```diff
@@ -92,17 +54,36 @@ class EscapedExpression extends Expression {
   }
 }
 
+/// A constant [Expression].
```

Comment:
How about we deprecate the Constant constructor and create two new constructors.

I.e. `Constant.string()` and `Constant.bool()`?

This would give us type safety and compile time checks to avoid the mistakes of inserting invalid data types.